### DISCUSSION TITLE
Is it possible for a DOM element to live in the light DOM (say, as a child of the body tag) but have it rendered into a LitElement?

### DISCUSSION THREAD

**Discussion by andyjessop** [lit/lit#4393] [General]
I'm trying to use editor.js, which has an unfortunate attribute of adding things like styles and tooltips into the head and body tags, instead of them living next to (or inside the editor root). 

See here for more discussion: https://github.com/codex-team/editor.js/discussions/1881

(There is no satisfactory solution over there yet)

I'm wondering if this community has any ideas. All I could think of was that the editor element would live in the light DOM and would, like a Portal in React, be rendered into a shadow root.

Is this feasible, or does anyone have any other ideas?

**Comment by justinfagnani** [comment]
You can use slots for this. Render a `<slot>` in your element's template, then put the element holding the editor's DOM into the light DOM of the element.
